src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
@stipsan Yes, I'm aware of it. Don't think if would be a problem because that's the last condition in the chain of possible options (after the `.yml`). See, it doesn't match any files in the project root. It matches files that are explicitly specified in `codeception.yml`, so if user want to specify file named `.envy` there's no reason to deny it in this case, I think.